### PR TITLE
Finish timeOffsetWrongCount

### LINKED ISSUE #1433: BMProto.timeOffsetWrongCount unfinished
Forgot to finish it after moving to asyncore. There should be a check for a threshold, and a panel notification.

### PR DESCRIPTION
**Pull request by PeterSurda**
- now notifies in status bar
- fixes #1433

### CODE REVIEW

File: src/network/tcp.py [hunk 1]
```diff
@@ -124,6 +127,23 @@ def antiIntersectionDelay(self, initial=False):
...
+                maximumTimeOffsetWrongCount and \
+                not self.fullyEstablished:
+            UISignalQueue.put((
+                'updateStatusBar',
+                # pylint: disable=protected-access
+                tr._translate(
+                    "MainWindow",
+                    "The time on your computer, %1, may be wrong. "
+                    "Please verify your settings."
+                ).arg(datetime.datetime.now().strftime("%H:%M:%S"))))
```
Comment (g1itch) [inline src/network/tcp.py:145]: Maybe use `l10n.formatTimestamp()` here, like in [class_singleWorker](../../blob/v0.6/src/class_singleWorker.py#L677)?

File: src/network/tcp.py [hunk 2]
```diff
@@ -124,6 +127,23 @@ def antiIntersectionDelay(self, initial=False):
...
+        """
+        Check if we have connected to too many nodes which have too high
+        time offset from us
+        """
+        if BMProto.timeOffsetWrongCount > \
+                maximumTimeOffsetWrongCount and \
+                not self.fullyEstablished:
+            UISignalQueue.put((
+                'updateStatusBar',
+                # pylint: disable=protected-access
```
Comment (g1itch) [inline src/network/tcp.py:140]: No need. Just `from tr import _translate`.

Comment (PeterSurda) [reply src/network/tcp.py:140]: ok

File: src/network/tcp.py (hunk 1 above)
Comment (PeterSurda) [reply src/network/tcp.py]: ok

**Review by g1itch — APPROVED** [review]
Looks good.

**Pull request merged**